fixture/vcr_cassettes/twilio_sms_from_success.json

```diff
@@ -13,23 +13,25 @@
     },
     "response": {
       "binary": false,
-      "body": "{\"sid\": \"resource-id\", \"date_created\": \"Wed, 25 Apr 2018 21:11:33 +0000\", \"date_updated\": \"Wed, 25 Apr 2018 21:11:33 +0000\", \"date_sent\": null, \"account_sid\": \"fake-account-sid\", \"to\": \"+15557654321\", \"from\": \"+15551234567\", \"messaging_service_sid\": null, \"body\": \"Hello there\", \"status\": \"queued\", \"num_segments\": \"1\", \"num_media\": \"0\", \"direction\": \"outbound-api\", \"api_version\": \"2010-04-01\", \"price\": null, \"price_unit\": \"USD\", \"error_code\": null, \"error_message\": null, \"uri\": \"/2010-04-01/Accounts/fake-account-sid/Messages/resource-id.json\", \"subresource_uris\": {\"media\": \"/2010-04-01/Accounts/fake-account-sid/Messages/resource-id/Media.json\"}}",
+      "body": "{\"body\": \"Hello there\", \"num_segments\": \"1\", \"direction\": \"outbound-api\", \"from\": \"+15551234567\", \"date_updated\": \"Tue, 06 Dec 2022 00:26:44 +0000\", \"price\": null, \"error_message\": null, \"uri\": \"/2010-04-01/Accounts/fake-account-sid/Messages/resource-id.json\", \"account_sid\": \"fake-account-sid\", \"num_media\": \"0\", \"to\": \"+15557654321\", \"date_created\": \"Tue, 06 Dec 2022 00:26:44 +0000\", \"status\": \"queued\", \"sid\": \"SM6fe74e301fe0b839d00820c31dca724e\", \"date_sent\": null, \"messaging_service_sid\": null, \"error_code\": null, \"price_unit\": \"USD\", \"api_version\": \"2010-04-01\", \"subresource_uris\": {\"media\": \"/2010-04-01/Accounts/fake-account-sid/Messages/resource-id/Media.json\"}}",
```

Comment:
Can we update the `sid` value to something fake, just to make it clear it's a test value. Maybe something like `SM123`?